index.html

```diff
@@ -1,19 +1,147 @@
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <a class="header-logo" href="#">
+          <img
+            class="logo-image"
+            src="./img/karma-logo.svg"
+            alt="logo of karma"
+          />
+        </a>
+        <nav class="nav">
+          <!-- (B) THE HAMBURGER -->
+          <label for="hamburger">&#9776;</label>
+          <input type="checkbox" id="hamburger" />
+          <ul class="nav-menu">
+            <li class="list-item">
+              <a href="./index.html" class="nav-link nav-active">Meet Karma</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">How it Works</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Store</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Blog</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Help</a>
+            </li>
+            <li class="list-item">
+              <a href="./order.html" class="nav-link">Login</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="home">
+        <div class="content">
+          <h2 class="intro-karma">Introducing Karma</h2>
+          <p class="slogan">Bring WiFi with you everywhere you go.</p>
+          <button class="learn-more">Learn More</button>
+        </div>
+      </section>
+      <section class="products">
+        <h2>Everyone needs a little Karma.</h2>
+        <div class="product-cards">
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-devices.svg"
+              alt="devices-logo"
+            />
+            <p>Internet for all devices</p>
+          </article>
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-coffee.svg"
+              alt="coffee-logo"
+            />
+            <p>Boost your productivity</p>
+          </article>
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-refill.svg"
+              alt="refill-logo"
+            />
+            <p>Pay as you go</p>
+          </article>
+        </div>
+      </section>
+      <section class="quoto-section">
+        <img
+          class="quoto-image"
+          src="./img/homepage-feature.jpg"
+          alt="quoto image"
+        />
+        <article class="quoto-container">
+          <div class="quoto-content">
+            <p>
+              <span>“</span>Wherever I am, I just don’t worry about my
+              connection anymore!<span>”</span>
+            </p>
+            <button class="get-karma">Get Karma today</button>
+          </div>
+        </article>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-container">
```

Comment:
Same thing here - why these 2 elements? They wrap exactly the same elements. Remove the div and move all styles to the `footer`.